src/HonzaBotner.Discord.Services/EventHandlers/ReminderReactionsHandler.cs

```diff
@@ -24,21 +25,19 @@ public ReminderReactionsHandler(IOptions<ReminderOptions> options, IRemindersSer
 
         public async Task<EventHandlerResult> Handle(MessageReactionAddEventArgs arguments)
         {
-            var emoji = arguments.Emoji;
-            var validEmojis = new[] { _options.CancelEmojiName, _options.JoinEmojiName };
-
-            if (!validEmojis.Contains(emoji))
-            {
-                return EventHandlerResult.Continue;
-            }
-
             if (arguments.User.IsBot)
             {
                 return EventHandlerResult.Stop;
             }
 
-            var reminder = await _service.GetByMessageIdAsync(arguments.Message.Id);
+            DiscordEmoji emoji = arguments.Emoji;
+            string[] validEmojis = { _options.CancelEmojiName, _options.JoinEmojiName };
+            if (!validEmojis.Contains(emoji))
+            {
+                return EventHandlerResult.Continue;
+            }
 
+            Reminder? reminder = await _service.GetByMessageIdAsync(arguments.Message.Id);
```

Comment:
Is there any reason for not using `var`? (as suggested by ReSharper)

Reply:
Will we rethink using vars, @ostorc?

Reply:
I am used to following .NET Runtime code guidelines which allows var only in specific cases, but we can probably go with vars...